Formula/d/dumbpipe.rb

```diff
@@ -0,0 +1,43 @@
+class Dumbpipe < Formula
+  desc "Unix pipes between devices"
+  homepage "https://dumbpipe.dev"
+  url "https://github.com/n0-computer/dumbpipe/archive/refs/tags/v0.28.0.tar.gz"
+  sha256 "bb7bd90eacebe505f2c669e4e13dac57c43c9c0eb5eca94dfa1378fd7cdcda84"
+  license any_of: ["MIT", "Apache-2.0"]
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    read, write = IO.pipe
+
+    listener_pid = spawn bin/"dumbpipe", "listen",
+                         out: write, err: write
```

Comment:
Redirecting both `stdout` and `stderr` to `write` seems weird here. Is that intentional?

Reply:
Ah, I see. I misunderstood the return value of `IO.pipe` here. Never mind.